test/e2e/common/deploy.go

```diff
@@ -147,19 +145,41 @@ func DeployCSI(f *framework.Framework, additionalInstallArgs string) (func(), er
 	cleanup := func() {
 		if BMDriverTestContext.CompleteUninstall {
 			CleanupLoopbackDevices(f)
-			// delete resources with finalizers
-			// pvcs and volumes are namespaced resources and deleting with it
-			deleteCSIResources(cmdExecutor, []string{"lvgs", "csibmnodes"})
+			// delete resources with finalizers and wait until node- and lvgcontroller reconcile requests
+			removeCRs(f, CsibmnodeGVR, LVGGVR)
+			deadline := time.Now().Add(30 * time.Second)
+			for {
+				time.Sleep(2 * time.Second)
+				if !isCRInstancesExists(f, CsibmnodeGVR) && !isCRInstancesExists(f, LVGGVR) {
+					break
+				}
+				if time.Now().After(deadline) {
+					e2elog.Logf("All csibmnodes or lvgs have not been deleted yet")
```

Comment:
Not _all_, but _some_. Please print CRs here for troubleshooting.

Reply:
fixed